Makefile

```
@@ -33,54 +37,29 @@ sls-only-deploy: # Deploys all lambdas - mandatory: PROFILE, VERSION=[commit has
 
 undeploy: # Undeploys whole project - mandatory: PROFILE
 	make serverless-remove VERSION="any"
-	make terraform-destroy-auto-approve STACKS=lambda-security-group
 
-build-and-deploy: # Builds and Deploys whole project - mandatory: PROFILE
+build-and-deploy: # - mandatory: PROFILE
 	make build VERSION=$(BUILD_TAG)
 	make push-images VERSION=$(BUILD_TAG)
 	make deploy VERSION=$(BUILD_TAG)
 
-populate-deployment-variables:
-	if [ "$(PROFILE)" == "demo" ] || [ "$(PROFILE)" == "live" ] || [ "$(PROFILE)" == "dev" ]; then
-		eval "$$(make aws-assume-role-export-variables)"
-		echo "export DOS_API_GATEWAY_USERNAME=$$(make -s secret-get-existing-value NAME=$(DOS_DEPLOYMENT_SECRETS) KEY=$(DOS_API_GATEWAY_USERNAME_KEY))"
-		echo "export DOS_API_GATEWAY_PASSWORD=$$(make -s secret-get-existing-value NAME=$(DOS_DEPLOYMENT_SECRETS) KEY=$(DOS_API_GATEWAY_PASSWORD_KEY))"
-	fi
-
 python-requirements: # Installs whole project python requirements
 	make docker-run-tools \
 		CMD="pip install -r requirements.txt -r requirements-dev.txt -r event_sender/requirements.txt" \
 		DIR=./application
 
 unit-test: # Runs whole project unit tests
```

Please revert this make target
```suggestion
unit-test: # Runs whole project unit tests
	make -s docker-run-tools \
	CMD="python -m pytest --cov=." \
This conversation was marked as resolved by JackPlowman
 Show conversation
	DIR=./application \
	ARGS=" \
		-e POWERTOOLS_LOG_DEDUPLICATION_DISABLED="1" \
		--volume $(APPLICATION_DIR)/event_sender:/tmp/.packages/event_sender \
		"```